clang-tools-extra/test/clang-doc/test-path-abs.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: echo "CHECK: var RootPath = \"%/t/docs\";" > %t/check.txt
+// RUN: cp "%s" "%t/test.cpp"
```

Comment:
I see you're trying to work around the path inconsistency issues, but that's just an end-around our testing practices, and we shouldn't be doing it. Tests server as documentation, and generating a test w/ `CHECK` lines undermines that pretty significantly. IMO if the test isn't useful w/ the regular expression, it isn't useful. As it stands I also can't reason about how the behavior here is changing as a result of this patch, so I don' think this is a particularly good/useful check.

If possible, it would be good to add these tests in another patch(e.g. pre-commit them testing the current behavior), and then update these to make them pass. That requires either stacking the PR using [spr](https://github.com/spacedentist/spr), or just managing the github PRs carefully yourself. Either is fine.